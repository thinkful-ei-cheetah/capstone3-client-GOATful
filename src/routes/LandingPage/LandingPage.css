.landing-container {
  display: flex;
  flex-flow: column wrap;
  align-items: center;
}

.hero {
  background: url('../../img/sfr.jpg');
  background-size: cover;
  background-position: center;
  padding: 4rem 2rem;
  display: grid;
  height: 30vh;
}

.red {
  color: rgb(13, 22, 43);
  font-weight: 900;
  font-size: 28px;
}

.hero > * {
  color: white;
}

.click-through, .AB-testing, .about {
  max-width: 850px;
  margin: 1.5rem;
  padding: 2rem;
  border: 1px solid #E2E2E2;
  border-radius: .5rem;
}

.click-through > figure, .AB-testing > figure, .about > figure {
  margin: auto;
  max-height: 400px;
  max-width: 600px;
  overflow: hidden;
  position: relative;
}

.click-through > figure > img, .AB-testing > figure > img, .about > figure > img {
  width: 100%;
}

.btn-container {
  display: flex;
  justify-content: center;
  padding-bottom: 1rem;
}