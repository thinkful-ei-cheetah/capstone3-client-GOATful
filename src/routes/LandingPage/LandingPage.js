import React, { Component } from 'react'
import './LandingPage.css'
import './laconic-bg-2.jpg'
import './dark-hero-bg.png'
import './aside-bg.png'
import DemoOnTable from './demo-bg.png'

class Landing extends Component {
  pushToLogin = () => {
    this.props.history.push('/login')
  }
  render() {
    return (
      <>
      <main className="landing-page-container">
        <section className='hero page'>
          <div className='hero-content'>
            <h2>Youtube Creators</h2>
            <h3>Ready to 10x your view counts?</h3>

            <p>Laconic is the only tool that helps you optimize the look and feel of your video thumbnails before you post to YouTube.</p>
            <button onClick={this.pushToLogin} className='create-laconic'>
              Create with Laconic
            </button>
          </div>
        </section>
        <section className='about'>
          <div className='left'>
            <div className="AboutText">
              <h3>
                What We Do
              </h3>
              <p className="p">
                Laconic allows you to see what your YouTube thumbnails will look like before you publish them. Save and compare multiple thumbnails related to a video. When you decide which thumbnail to use, copy the fields directly from Laconic and publish your video!  
            </p>
              <div className="btn-container">
                <button onClick={this.pushToLogin} className='create-laconic'>
                  Create with Laconic
                </button>
              </div>
            </div>
          </div>
          <div className='right'>
            <img src={DemoOnTable} alt='Laconic on mac'/>
          </div>
        </section>
        
        <aside>
          <div className='quote'>
            Since using Laconic my total views and subscriptions have sky rocketed!
          </div>
        </aside>
        
      </main>
      </>
    );
  }
}

export default Landing