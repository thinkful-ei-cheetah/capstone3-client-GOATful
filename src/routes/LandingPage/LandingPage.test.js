import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter } from 'react-router-dom';
import LandingPage from './LandingPage';
import renderer from 'react-test-renderer';

describe('LandingPage Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
    <BrowserRouter>
      <LandingPage />
    </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<LandingPage/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
})
