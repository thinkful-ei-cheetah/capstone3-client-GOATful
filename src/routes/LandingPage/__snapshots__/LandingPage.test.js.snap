// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage Component renders the UI as expected 1`] = `
Array [
  <section
    className="hero page"
  >
    <h1>
      Let your thumbnail do the talking.
    </h1>
    <div>
      <h2
        className="red"
      >
        With Laconic
      </h2>
    </div>
  </section>,
  <div
    className="landing-container"
  >
    <section
      className="click-through"
    >
      <figure>
        <img
          alt="Click through rate"
          src="https://www.lyfemarketing.com/blog/wp-content/uploads/2018/06/what-is-CTR.jpg"
        />
      </figure>
      <h3>
        So What Is Click-Through Rate, Anyway?
      </h3>
      <p>
        is the rate at which your videos are clicked. This number is the percentage of people who view your video (impressions) and then actually go on to click the video (clicks). The formula for CTR looks like this:
      </p>
      <h4>
        (Total Clicks on Video) / (Total Impressions) = Click Through Rate
      </h4>
      <p>
        Generally, you can view your click-thru rate within the dashboard of your YouTube account. A high CTR means that a high percentage of people who see your video click it.
      </p>
    </section>
    <section
      className="AB-testing"
    >
      <figure>
        <img
          alt="A/B Testing"
          src="https://mzdpd1mg35f29101q1lbu25g-wpengine.netdna-ssl.com/wp-content/uploads/2019/01/AB-guide-small.png"
        />
      </figure>
      <h3>
        WHAT IS A/B TESTING
      </h3>
      <p>
        An “A/B Test” is a test where 2 variants of something (anything) is tested for equal amounts of time, to see which version performs the best. In this case, what is being tested and compared is 2 versions of a thumbnail, of your own choosing. By uploading 2 thumbnails into the system, they are changed each day automatically on your channel (for a chosen period), and by the end of the test we automatically gather the data from your YouTube Analytics and see which of the thumbnails resulted in the most views.
      </p>
    </section>
    <section
      className="about"
    >
      <figure>
        <img
          alt="YouTube"
          src="https://images.unsplash.com/photo-1521302200778-33500795e128?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1500&q=80"
        />
      </figure>
      <h3>
        What we do
      </h3>
      <p>
        We Preview your thumbnails, so you can see what they will look like in Youtube's styling in both desktop and App view. Save your thumbnails so you can easily see the difference between the two. Keep all your thumbnail Ideas in one place!
      </p>
    </section>
    <div
      className="btn-container"
    >
      <button
        className="create-laconic"
        onClick={[Function]}
      >
        Create with Laconic
      </button>
    </div>
  </div>,
]
`;
